web/client/utils/GeoStoryUtils.js

```diff
@@ -94,7 +94,10 @@ export const getClassNameFromProps = ({ theme = {}, align = 'center', size = 'fu
  * @prop {string} theme.value style key
  * @prop {string} theme[theme.value] a style object referred to the style key
  */
-export const getThemeStyleFromProps = ({ theme = {} }) => {
+export const getThemeStyleFromProps = ({ theme = {}, storyTheme }) => {
+    if (theme?.value === '') {
```

Comment:
looking at the code the theme property can be represented by a string or an object so the default one could match one of these statements:

theme === ''
theme === undefined
theme?.value === ''
theme?.value === undefined